modules/text/test/test_detection_swt.cpp

```diff
@@ -0,0 +1,75 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+#include "opencv2/imgcodecs.hpp"
+
+using namespace cv;
+using namespace std;
+namespace opencv_test { namespace {
+
+TEST (TextDetectionSWT, accuracy_light_on_dark) {
+    std::vector<cv::Rect> letters;
+    String dataPath = cvtest::TS::ptr()->get_data_path() + "cv/mser/mser_test.png";
```

Comment:
Please use `cvtest::findDataFile()` instead (see `test_detection.cpp`)

You may use images from samples directory too.

Reply:
Having issues with this. It doesn't work on my build, not even in ```test_detection.cpp```. I will try to debug